### DISCUSSION TITLE
24–28 October

### DISCUSSION THREAD

**Discussion by esheehan-gsl** [NOAA-GSL/unified-graphics#114] [Progress Report]
🚴🏻 **Progress**

- Improve loading status indicator and rendering performance (#110)

😖 **Regressions**

None

💬 **To Discuss**

None

📆 **Next Week**

- Demo
- Quantized color scales and legend for heat map (#106)

**Comment by ian-noaa** [comment]
🚴🏻 **Progress**

- Initial capabilities to read data from S3 (#80)

😖 **Regressions**

- Fixed a security issue that was preventing us from deploying to VLab (#111)

💬 **To Discuss**

 - None

📆 **Next Week**

- Demo
- #67 
- Explore utilizing real-time RRFS/RTMA data from the NODD S3 bucket
- Work with VLab on accessing production logs and on ways to easily make changes to the Kubernetes manifests in the production cluster
